interactions/api/dispatch.py

```diff
@@ -1,15 +1,11 @@
 from asyncio import get_event_loop
-from logging import Logger, StreamHandler, basicConfig, getLogger
+from logging import Logger, basicConfig, getLogger
 from typing import Coroutine, Optional
 
-from ..base import CustomFormatter, Data
+from ..base import Data
 
 basicConfig(level=Data.LOGGER)
```

Comment:
basicConfig should also be removed? (i think)

Reply:
yep, im a moron and forgot that would also enable a stream